packages/services/api/src/modules/policy/utils.ts

```diff
@@ -0,0 +1,64 @@
+import type { SchemaPolicyApi } from '@hive/policy';
+import { TRPCClientError } from '@trpc/client';
+import { RuleInstanceSeverityLevel, SchemaPolicyInput } from '../../__generated__/types';
+
+export function formatTRPCErrors(e: TRPCClientError<SchemaPolicyApi>) {
+  if (e.data?.zodError) {
+    return {
+      error: {
+        __typename: 'UpdateSchmeaPolicyResultError' as const,
+        message: e.data.formatted || e.message,
+        code: 'VALIDATION_ERROR',
+      },
+    };
+  }
+
+  return {
+    error: {
+      __typename: 'UpdateSchmeaPolicyResultError' as const,
+      message: e.message,
```

Comment:
We need to make sure to not leak any internal error messages

Reply:
@kamilkisiela the error in this case is coming from the `policy` service - it only throws validation errors for the policy config